bidwire/alembic/env.py

```diff
@@ -1,11 +1,16 @@
 from __future__ import with_statement
+import sys
+import os
+
 from alembic import context
 from sqlalchemy import engine_from_config, pool
 from logging.config import fileConfig
+sys.path.append(os.path.dirname(os.path.dirname(__file__)))
+from bid import Base
```

Comment:
@klertmen FYI see what Kushi did here to add the directory of the current fie to the python path, so that she can then import `bid`. Likely this approach would work too in a migration itself. (So we can use our code in the migrations themselves.)